### PR TITLE
GOV MODE SENSOR

### PR DESCRIPTION
**Pull request by robthomson**
Added sensor 0x5450 to send governor status to frsky as

0 - OFF
1 - IDLE
2 - SPOOLUP
3 - RECOVERY
4 - ACTIVE
5 - THR-OFF
6 - LOST-HS
7 - AUTOROT
8 - BAILOUT
100 - DISABLED
101 - DISARMED

### CODE REVIEW

File: src/main/telemetry/smartport.c [hunk 1]
```diff
@@ -214,6 +217,11 @@ static smartPortWriteFrameFn *smartPortWriteFrame;
 static bool smartPortMspReplyPending = false;
 #endif
 
+
+
+
+
+
```
Comment (rotorflight) [inline src/main/telemetry/smartport.c:224]: Please don't add unnecessary empty line

File: src/main/telemetry/smartport.c [hunk 2]
```diff
@@ -426,6 +437,8 @@ static void initSmartPortSensors(void)
         ADD_SENSOR(FSSP_DATAID_ADJVALUE);
     }
 
+
+
```
Comment (rotorflight) [inline src/main/telemetry/smartport.c:441]: Please remove unnecessary empty lines

File: src/main/telemetry/smartport.c [hunk 3]
```diff
@@ -130,6 +132,7 @@ enum
     FSSP_DATAID_ADJFUNC    = 0x5110 , // custom
     FSSP_DATAID_ADJVALUE   = 0x5111 , // custom
     FSSP_DATAID_CAP_USED   = 0x5250 ,
+    FSSP_DATAID_GMODE      = 0x5450 , //custom
```
Comment (rotorflight) [inline src/main/telemetry/smartport.c:135]: GMODE is not very descriptive. Maybe GOV_MODE

**Review by rotorflight — CHANGES_REQUESTED** [review]
Please see the comments

File: src/main/telemetry/smartport.c (hunk 1 above)
Comment (robthomson) [reply src/main/telemetry/smartport.c]: Done

File: src/main/telemetry/smartport.c (hunk 3 above)
Comment (robthomson) [reply src/main/telemetry/smartport.c]: Done

**Comment by robthomson** [comment]
Should all be ok now

File: src/main/telemetry/smartport.c [hunk 4]
```diff
@@ -130,6 +132,7 @@ enum
     FSSP_DATAID_ADJFUNC    = 0x5110 , // custom
     FSSP_DATAID_ADJVALUE   = 0x5111 , // custom
     FSSP_DATAID_CAP_USED   = 0x5250 ,
+    FSSP_DATAID_GOV_MODE      = 0x5450 , //custom
```
Comment (rotorflight) [inline src/main/telemetry/smartport.c:135]: Can you keep the = aligned...

File: src/main/telemetry/smartport.c [hunk 5]
```diff
@@ -157,7 +160,7 @@ enum
 };
 
 // if adding more sensors then increase this value (should be equal to the maximum number of ADD_SENSOR calls)
-#define MAX_DATAIDS 24
+#define MAX_DATAIDS 26
```
Comment (rotorflight) [inline src/main/telemetry/smartport.c:163]: You added one sensor. Why increasing this +2 ?

**Review by rotorflight — CHANGES_REQUESTED** [review]
minor fix needed

File: src/main/telemetry/smartport.c [hunk 6]
```diff
@@ -331,6 +334,9 @@ static void initSmartPortSensors(void)
 {
     frSkyDataIdTableInfo.index = 0;
 
+    //prob need configurator option for these?
```
Comment (rotorflight) [inline src/main/telemetry/smartport.c:337]: That's right, you need to add a telemetry "sensor" for this.

**Review by rotorflight — CHANGES_REQUESTED** [review]
Add a telemetry sensor enable for the gov status

**Review by rotorflight — APPROVED** [review]

**Pull request merged**